sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -155,6 +155,38 @@ def create_options(self):
         options.view_as(StandardOptions).streaming = True
       return options
 
+    def test_pardo_state_with_custom_key_coder(self):
```

Comment:
should we add this the `portable_runner_test.py` and ensure that all runners are executing it (and possibly skipping it if they don't support it).

Reply:
Yep, done.